src/internal/streamstore/fault_errors.go

```diff
@@ -0,0 +1,67 @@
+package streamstore
+
+import (
+	"context"
+
+	"github.com/alcionai/clues"
+	"github.com/alcionai/corso/src/internal/kopia"
+	"github.com/alcionai/corso/src/pkg/fault"
+	"github.com/alcionai/corso/src/pkg/path"
+	"github.com/pkg/errors"
```

Comment:
rip linter not catching this T_T
```suggestion
	"github.com/alcionai/clues"
	"github.com/pkg/errors"
	
	"github.com/alcionai/corso/src/internal/kopia"
	"github.com/alcionai/corso/src/pkg/fault"
	"github.com/alcionai/corso/src/pkg/path"
```

Reply:
😭

Reply:
Any thoughts on running `make fmt` in the CI and seeing if it produces any diffs? The linter seem to not do its job from time to time.